-code-checks

ERROR: Build errors found during clang-tidy run.
```
SimTransport/HectorProducer/src/HectorProducer.cc:123:7: error: use of undeclared identifier 'LogVerbatim'; did you mean 'edm::LogVerbatim'? [clang-diagnostic-error]
      LogVerbatim("HectorEventProcessing") << "Hector correspondence table: " << (*NewCorrespondenceMap)[i];
      ^
/cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc820/cms/cmssw-patch/CMSSW_11_2_X_2020-08-24-1100/src/FWCore/MessageLogger/interface/MessageLogger.h:325:9: note: 'edm::LogVerbatim' declared here
--
gmake: *** [config/SCRAM/GMake/Makefile.coderules:128: code-checks] Error 2
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
```

--- a/SimPPS/PPSSimTrackProducer/plugins/PPSSimTrackProducer.cc
+++ b/SimPPS/PPSSimTrackProducer/plugins/PPSSimTrackProducer.cc
@@ -146,8 +146,8 @@ void PPSSimTrackProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSe
   //theTransporter->clear();
   theTransporter->process(evt, iSetup, engine);
 
-  //  if (m_verbosity)
-  evt->print();
+  if (m_verbosity)
+    evt->print();
 
   unique_ptr<HepMCProduct> newProduct(new edm::HepMCProduct());
   newProduct->addHepMCData(evt);
@@ -160,7 +160,6 @@ void PPSSimTrackProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSe
 
   if (m_verbosity) {
     for (unsigned int i = 0; i < (*NewCorrespondenceMap).size(); ++i)
-      //edm::LogVerbatim("ProtonTransportEventProcessing")
       LogDebug("ProtonTransportEventProcessing")
           << "ProtonTransport correspondence table: " << (*NewCorrespondenceMap)[i];
   }

--- a/SimTransport/HectorProducer/src/Hector.cc
+++ b/SimTransport/HectorProducer/src/Hector.cc
@@ -161,17 +161,6 @@ void Hector::clear() {
   m_isCharged.clear();
 }
 
-/*
-  bool Hector::isCharged(const HepMC::GenParticle * p){
-  const ParticleData * part = pdt->particle( p->pdg_id() );
-  if (part){
-  return part->charge()!=0;
-  }else{
-  // the new/improved particle table doesn't know anti-particles
-  return  pdt->particle( -p->pdg_id() )!=0;
-  }
-  }
-*/
 void Hector::add(const HepMC::GenEvent *evt, const edm::EventSetup &iSetup) {
   H_BeamParticle *h_p;
   unsigned int line;
@@ -224,7 +213,6 @@ void Hector::add(const HepMC::GenEvent *evt, const edm::EventSetup &iSetup) {
           m_pz[line] = (*eventParticle)->momentum().pz();
 
           if (m_verbosity) {
-            //            LogDebug("HectorEventProcessing") << "Hector:add: barcode = " << line << " status = " << g->status()
             edm::LogVerbatim("HectorEventProcessing")
                 << "Hector:add: barcode = " << line << " status = " << g->status() << " PDG Id = " << g->pdg_id()
                 << " mass = " << mass << " pz = " << pz << " charge = " << charge
@@ -340,15 +328,12 @@ void Hector::filterZDC(TRandom3 *rootEngine) {
     for (it = m_beamPart.begin(); it != m_beamPart.end(); ++it) {
       line = (*it).first;
       part = (*it).second;
-      //if (m_verbosity) {
-      //       LogDebug("HectorEventProcessing")
-      edm::LogVerbatim("HectorEventProcessing")
-          << "Hector:filterZDC: barcode = " << line << " charge  = " << (*m_isCharged.find(line)).second;
-      if (m_FP420Transport)
-        LogDebug("HectorEventProcessing") << " isStoppedFP420 =" << (*m_isStoppedfp420.find(line)).second;
-      //}
-      //      if ( ((*m_isStoppedfp420.find(line)).second) &&
-      //      ((*m_isCharged.find(line)).second) ) {
+      if (m_verbosity) {
+        edm::LogVerbatim("HectorEventProcessing")
+            << "Hector:filterZDC: barcode = " << line << " charge  = " << (*m_isCharged.find(line)).second;
+        if (m_FP420Transport)
+          LogDebug("HectorEventProcessing") << " isStoppedFP420 =" << (*m_isStoppedfp420.find(line)).second;
+      }
       if (((*m_isCharged.find(line)).second)) {
         if (m_verbosity)
           LogDebug("HectorEventProcessing") << "Hector:filterZDC: barcode = " << line << " propagated ";
@@ -429,13 +414,12 @@ void Hector::filterD1(TRandom3 *rootEngine) {
     for (it = m_beamPart.begin(); it != m_beamPart.end(); ++it) {
       line = (*it).first;
       part = (*it).second;
-      //  if (m_verbosity)
-      //        LogDebug("HectorEventProcessing")
-      edm::LogVerbatim("HectorEventProcessing")
-          << "Hector:filterD1: barcode = " << line << " isStoppedZDC =" << (*m_isStoppedzdc.find(line)).second;
+      if (m_verbosity)
+        edm::LogVerbatim("HectorEventProcessing")
+            << "Hector:filterD1: barcode = " << line << " isStoppedZDC =" << (*m_isStoppedzdc.find(line)).second;
       if (((*m_isStoppedzdc.find(line)).second) || !((*m_isCharged.find(line)).second)) {
-        //if (m_verbosity)
-        edm::LogVerbatim("HectorEventProcessing") << "Hector:filterD1: barcode = " << line << " propagated ";
+        if (m_verbosity)
+          edm::LogVerbatim("HectorEventProcessing") << "Hector:filterD1: barcode = " << line << " propagated ";
 
         direction = (*m_direct.find(line)).second;
         if (m_verbosity)

--- a/SimTransport/HectorProducer/src/HectorProducer.cc
+++ b/SimTransport/HectorProducer/src/HectorProducer.cc
@@ -102,9 +102,8 @@ void HectorProducer::produce(edm::Event &iEvent, const edm::EventSetup &es) {
     m_Hector->filterD1(rootEngine);
   }
   evt_ = m_Hector->addPartToHepMC(evt_);
-  //if (m_verbosity) {
-  evt_->print();
-  //}
+  if (m_verbosity)
+    evt_->print();
 
   edm::LogVerbatim("SimTransportHectorProducer") << "new HepMC product ";
 
@@ -118,10 +117,10 @@ void HectorProducer::produce(edm::Event &iEvent, const edm::EventSetup &es) {
   edm::LHCTransportLinkContainer thisLink(m_Hector->getCorrespondenceMap());
   (*NewCorrespondenceMap).swap(thisLink);
 
-  //  if (m_verbosity) {
-  for (unsigned int i = 0; i < (*NewCorrespondenceMap).size(); ++i)
-    edm::LogVerbatim("HectorEventProcessing") << "Hector correspondence table: " << (*NewCorrespondenceMap)[i];
-  // }
+  if (m_verbosity) {
+    for (unsigned int i = 0; i < (*NewCorrespondenceMap).size(); ++i)
+      edm::LogVerbatim("HectorEventProcessing") << "Hector correspondence table: " << (*NewCorrespondenceMap)[i];
+  }
 
   iEvent.put(std::move(NewCorrespondenceMap));
   edm::LogVerbatim("SimTransportHectorProducer") << "produce end ";

--- a/SimTransport/HectorProducer/src/HectorProducer.cc
+++ b/SimTransport/HectorProducer/src/HectorProducer.cc
@@ -120,8 +120,7 @@ void HectorProducer::produce(edm::Event &iEvent, const edm::EventSetup &es) {
 
   //  if (m_verbosity) {
   for (unsigned int i = 0; i < (*NewCorrespondenceMap).size(); ++i)
-    LogVerbatim("HectorEventProcessing") << "Hector correspondence table: " << (*NewCorrespondenceMap)[i];
-  //LogDebug("HectorEventProcessing") << "Hector correspondence table: " << (*NewCorrespondenceMap)[i];
+    edm::LogVerbatim("HectorEventProcessing") << "Hector correspondence table: " << (*NewCorrespondenceMap)[i];
   // }
 
   iEvent.put(std::move(NewCorrespondenceMap));

